.github/workflows/llm_performance_tests.yml

```diff
@@ -51,6 +51,7 @@ jobs:
         # pip install transformers_stream_generator for model internlm-chat-7b-8k
         # pip install tiktoken for model Qwen-7B-Chat-10-12
         run: |
+          python -m pip install --pre --upgrade bigdl-llm[xpu_2.1] -f https://developer.intel.com/ipex-whl-stable-xpu
           python -m pip install --upgrade pip
           python -m pip install --upgrade wheel
           python -m pip install --upgrade omegaconf
```

Comment:
We may install bigdl-llm after pip and wheel :)

Reply:
sure:)